service/src/main/java/bio/terra/cbas/dependencies/github/GitHubService.java

```diff
@@ -0,0 +1,61 @@
+package bio.terra.cbas.dependencies.github;
```

Comment:
there isn't an official GitHub API client, but there are some third party ones that I chose not to use. Should I favor those third party libraries over writing something like this class?